packages/web/src/pages/profile-page/components/mobile/ProfilePage.tsx

```diff
@@ -198,34 +190,128 @@ const collectiblesTab = {
 const artistTabsWithCollectibles = [...artistTabs, collectiblesTab]
 const userTabsWithCollectibles = [...userTabs, collectiblesTab]
 
-const getMessages = ({
-  name,
-  isOwner
-}: {
-  name: string
-  isOwner: boolean
-}) => ({
-  emptyTracks: isOwner
-    ? "You haven't created any tracks yet"
-    : `${name} hasn't created any tracks yet`,
-  emptyAlbums: isOwner
-    ? "You haven't created any albums yet"
-    : `${name} hasn't created any albums yet`,
-  emptyPlaylists: isOwner
-    ? "You haven't created any playlists yet"
-    : `${name} hasn't created any playlists yet`,
-  emptyReposts: isOwner
-    ? "You haven't reposted anything yet"
-    : `${name} hasn't reposted anything yet`
-})
-
 const g = withNullGuard((props: ProfilePageProps) => {
   const { profile } = props
   if (profile) {
     return { ...props, profile }
   }
 })
 
+const TracksTab = ({
+  isOwner,
+  profile,
+  handle,
+  getLineupProps
+}: {
+  isOwner: boolean
+  profile: User
+  handle: string
+  getLineupProps: (lineup: any) => any
+}) => {
+  const {
+    fetchNextPage,
+    play,
+    pause,
+    lineup,
+    isFetchingNextPage,
+    hasNextPage,
+    pageSize
+  } = useProfileTracks({
+    handle
+  })
+
+  if (profile.track_count === 0) {
+    return (
+      <EmptyTab
+        message={
+          <>
+            {isOwner
+              ? "You haven't created any tracks yet"
+              : `${profile.name} hasn't created any tracks yet`}
```

Comment:
I think this one is wrong. `EmptyTab` already does
```
  const text = props.isOwner
    ? `You haven't ${props.text} yet...`
    : `${props.name} hasn’t ${props.text} yet...`
```
so you're gonna get a double "You haven't"

(similar instance below)